corporate/finance/monthly-reconciliation.rst

```diff
@@ -0,0 +1,32 @@
+:hide-toc:
+:og:description: The corporation should perform monthly reconciliation and reporting of all financial transactions to ensure the board of directors has visibility into the financial health of the organization.
+
+Monthly reconciliation
+======================
+
+The corporation should perform monthly reconciliation and reporting of all financial transactions to ensure the board of directors has visibility into the financial health of the organization.
+
+.. vale Google.Passive = NO
+.. vale write-good.E-Prime = NO
+.. vale write-good.Passive = NO
+
+RoboJackets responsibilities
+----------------------------
+
+These tasks should be completed by end of day on the first day of the following month.
+
+* `Sync BILL Spend & Expense transactions to QuickBooks <https://help.bill.com/direct/s/article/6854501>`_
+* Sync `invoices <https://loop.robojackets.org/docs/quickbooks/invoices/>`_ and `payments <https://loop.robojackets.org/docs/quickbooks/payments/>`_ from Loop to QuickBooks
```

Comment:
Workday sync mention? or maybe in loop docs already

Reply:
It is covered in the Loop documentation here: https://loop.robojackets.org/docs/workday/data-load/